source/application/utils/File.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright(c) 2017 - Hourglass Resurrection Team
+ * Hourglass Resurrection is licensed under GPL v2.
+ * Refer to the file COPYING.txt in the project root.
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace Utils
+{
+    namespace File
+    {
+        enum class FileFilter
+        {
+            AllFiles,
+            HourglassMovie,
+            Executable,
+            Config,
+            WatchList,
+            AVI
+        };
+
+        std::string GetFileNameOpen(
+            const std::string location,
```

Comment:
You might've meant to pass all these by const reference, in which case you forgot the &.